cmd/grafana-agent/entrypoint.go

```diff
@@ -400,6 +416,20 @@ func (ep *Entrypoint) Start() error {
 		})
 	}
 
+	if ep.cfg.AgentManagement.Enabled {
+		managementContext, managementCancel := context.WithCancel(context.Background())
+		defer managementCancel()
+
+		// By this point ep.cfg.Validate() has already been called, so the configured time must be valid
+		sleepTime, _ := ep.cfg.AgentManagement.SleepTime()
+		t := time.NewTicker(sleepTime)
+		g.Add(func() error {
+			return ep.pollConfig(managementContext, t)
```

Comment:
Are we only setting the ticker interval once at the start, or is there a way to reset it? I couldn't find where this would happen at a glance.

Reply:
The initial intention was not to allow this, but perhaps if so this should be a required CLI flag instead of a config element? cc: @jcreixell

Reply:
We also may leave this for a separate followup PR